src/sentry/api/event_search.py

```diff
@@ -617,7 +617,16 @@ def visit_aggregate_duration_filter(self, node, children):
             # Even if the search value matches duration format, only act as
             # duration for certain columns
             function = resolve_field(search_key.name, self.params, functions_acl=FUNCTIONS.keys())
-            if function.aggregate is not None and self.is_duration_key(function.aggregate[1]):
+
+            is_duration_key = False
+            if function.aggregate is not None:
+                args = function.aggregate[1]
+                if isinstance(args, list):
+                    is_duration_key = all(self.is_duration_key(arg) for arg in args)
```

Comment:
I see, the new functions support more than one arg, is there anywhere else we need to support multiple arguments or just this?

Reply:
I don't see any other places that we need to add additional support for this right now. Additionally, this is going to be purely internal for now so I'm not worried that there will be any impact.